### ISSUE TITLE
Correspond selection from itinerary to descriptions in sidebar

### ISSUE THREAD

**Issue by GrosR** [clowdfish/ttg-frontend#84]
Once a flight selection is done. we should also add the information about the selected flight in the left pane below the flight selection bar. The following information should be applied: 
Line 1: Flight route, like: ZRH -> HAN

Like in the other screens, we should have more ,detailed information in a popup view: 
Line 1: Zürich Airport -> Hannover Airport
Line 2: Overall flight time
Line 3: Upgrade Information (None, Business Class, ...)
Line 4: Price

**optional for a later stage**
and an even more detailed lightbox screen on click. 

Information like above plus
- baggage information
- distance from departure to Zürich Airport
- distance from destination Airport to appointment
- weather map for appointment date
- etc.

**Comment by damianmartone** [comment]
The itinerary section has been redesigned in order to follow a different approach and be able to hold more relevant information

![screen shot 2016-07-29 at 11 22 45](https://cloud.githubusercontent.com/assets/8942032/17243813/e18b5bd0-557e-11e6-85ad-79b7a3c48a1f.png)

- The sidebar has been expanded one column more, to better display information
- There is now the difference between appointments / questions / Flights 
- The "done" state had some reinforcement with a different icon and green border
- The price has been removed from the appointment section, and will now appear alongside the selection of the user
![screen shot 2016-07-29 at 11 25 03](https://cloud.githubusercontent.com/assets/8942032/17243867/1ecfbb1c-557f-11e6-8e2e-b76c61993c88.png)
- This selection will automatically change in the "active" state when the user selects a different option (and the option will be highlighted with a green border)
- There is the chance that we have just 1 option for the itinerary, in that case we'll display it as a normal card
![screen shot 2016-07-29 at 11 26 20](https://cloud.githubusercontent.com/assets/8942032/17243896/4c3cc16c-557f-11e6-8c7c-39dc9c6b95c2.png)
- IF there are more options, the selection will be done inside the card, in the sub-table
![screen shot 2016-07-29 at 11 27 04](https://cloud.githubusercontent.com/assets/8942032/17243916/65d6efbc-557f-11e6-836c-2920d50e74a6.png)
- The selection will remain in the sidebar even in done states to remind the user of the choice and prices
![screen shot 2016-07-29 at 11 27 48](https://cloud.githubusercontent.com/assets/8942032/17243933/7f130d3a-557f-11e6-902a-44681905677a.png)
- The editing and change of location for each appointment will be done in another issue

**Comment by GrosR** [comment]
Hi Damian, looks good so far. Just a question for my understanding... I am not sure that I understood the process correctly, probably due to the chosen test data: 
- in the sample, you chose an appointment in the left pane that has a flight. The right pane appears by clicking "Change" in the appointment in the left pane. 
- As the appointment was booked with a flight, the previously selected flight will be highlighted on the right by default. 

In your sample, you selected a flight and display train details... I assume this is a sample data bug? 

The intended behavior is, that a user can click another route and this will update the left pane. In case the route has timing alternatives, these will be displayed and the default time chosen by us can be altered by selecting another timing. 
Once a timing alternative is selected (in case one is available at all), the left pane gets updated. 

Right?

**Comment by damianmartone** [comment]
No, sorry what you see there its not a real sample its the page that has all the styles together, but I can show you how it would be on a real case

![screen shot 2016-07-29 at 14 52 12](https://cloud.githubusercontent.com/assets/8942032/17248624/1179448e-559c-11e6-8946-cade18f39c56.png)

Here we have 3 appointments, one gap between appointments and a possible return flight.
The user has already selected to depart from Vienna (origin) to Zürich for his first meeting. He chose a train to go there in the previous step, and clicked next.
Now he is selecting how to go from Zürich to Berlin, he has 3 possible itineraries: two flight possibilities and a train, but the train has more than one schedule available.
Now he has selected the flight but he could also check for trains

![screen shot 2016-07-29 at 14 55 08](https://cloud.githubusercontent.com/assets/8942032/17248708/77797042-559c-11e6-9f05-59866ec59c93.png)
Here the user has tapped on the train card and can see the train options. By default there's one selected (the most convenient one) but the user can select the other one and then click next to go to the next appointment / question

the CHANGE button is not yet developed, but that is specifically to edit the appointment's details again, and re-calculate. Has nothing to do with changing flights or itineraries.

**Comment by GrosR** [comment]
thanks, got it. Where would the user change the departure location for this appointment in case he would need to change it? And where does he see that the departure starts from his latest appointment? 
Would this be done in the step before?

**Comment by damianmartone** [comment]
He would see where his departure starts cause the flight goes (DEPARTURE POINT) - (DESTINATION OF MEETING). If he clicks on CHANGE in the sidebar he can edit the appointment and change the departure city

**Comment by GrosR** [comment]
Sounds good. I see that the refinements really improve the workflow and the intuitive logic

**Comment by davidtran** [comment]
Can he change the appointment location ?

**Comment by GrosR** [comment]
Sure. That is was Damian wrote already

**Comment by damianmartone** [comment]
@GrosR @davidtran here is the final screen for flight selection, the hotel one is in the thread for hotel selection in this issue list
![flight](https://cloud.githubusercontent.com/assets/8942032/17301367/e7bd21ce-5816-11e6-98b9-d7c8a101df2f.png)
